src/App.css

```diff
@@ -1,16 +1,10 @@
-.App {
-  text-align: center;
-}
-
```

Comment:
- [ ] It would be nice if you use the [latest microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md) because you are using the old version of the readme file you have some aspects of the [latest readme file](https://github.com/microverseinc/readme-template/blob/master/README.md) but you are missing the whole layout which has navigations and organized content :thinking: Kindly use the readme template properly :+1:

Reply:
- [ ] this read-me file it was the latest please check again and this is optional suggestion.

Reply:
This is not an optional suggestion. Having a descriptive readme file is a mandatory requirement for each project and we check it in each milestone because you have this requirement in your each milestone requirements as you can see below :arrow_down:

![image](https://user-images.githubusercontent.com/93039359/222051345-1bfa52fe-08cd-40b0-9721-d723326eb321.png)